src/ontology/reports/icd10who.subclass.added.robot.tsv

```diff
@@ -0,0 +1,20 @@
+subject_mondo_id	subject_mondo_label	object_mondo_id	subject_source_id	object_source_id	object_mondo_label
+ID	LABEL	SC %			
+MONDO:0005066	metabolic disease		ICD10WHO:E70-E90	ICD10WHO:IV	
```

Comment:
`object_mondo_id` still missing.

Reply:
@matentzn I haven't yet re-run these outputs and am doing that today. I could probably hold off on checking off the checkbox, or add another checkbox for 'rerun outputs'.

Reply:
Just re-ran. The outputs look alright to my glancing.

ICD10WHO changed in that the `.added` output is now empty; this is because all of its `object_mondo_id`s were missing, so these new terms are supposedly not yet in Mondo, so an empty table for it is now expected.

Reply:
There are no ICD10WHO mappings in mondo.sssom.tsv?